jujugui/static/gui/src/app/components/usso-login-link/test-usso-login-link.js

```diff
@@ -36,11 +36,18 @@ describe('USSOLoginLink', () => {
           displayType={'text'}
           loginToController={sinon.stub()} />);
     assert.deepEqual(output,
-      <a className="logout-link"
-        onClick={output.props.onClick}
-        target="_blank">
-        Login
-      </a>);
+      <div className="usso-login">
+        <a className="logout-link"
+          onClick={output.props.onClick}
+          target="_blank">
+          Login
+        </a>
+        <div className="usso-login__notification">
+          When requested,
+          in your address bar above, please allow popups
+          from {window.location.origin} to login.
```

Comment:
Logout uses a different mechanism (clears the local state and redirects the current window)